plugins/thunderbolt/thunderbolt.conf

```diff
@@ -7,3 +7,6 @@ MinimumKernelVersion=4.13.0
 
 # Forces delaying activation until shutdown/logout/reboot
 DelayedActivation=false
+
+# Disable device from plugin backend
+DeniedDevice=00d4b071
```

Comment:
Don't add this - as discussed, try to inhibit from dell-dock plugin instead.

Reply:
thanks for the catch, missed to remove this test.